Test.hs

```diff
@@ -22,7 +22,7 @@ processOutput p args = Text.strip . Text.pack <$> readCreateProcess ((proc (Text
 -- * OpenGL
 
 -- | Returns the path to the nixGLXXX binary.
-getNixGLBin version = (<>("/bin/"<>version)) <$> processOutput "nix-build" ["./", "-A", version, "--arg", "pkgs", "import (fetchTarball " <> currentChannel <> ")"]
+getNixGLBin version = (<>("/bin/"<>version)) <$> processOutput "nix-build" ["./", "-A", version, "--argstr", "nixpkgs", currentChannel]
```

Comment:
Should be rebased on top of #39

Reply:
The `fetchTarball` disappears and it still fails.

Reply:
fixed!